src/main/webapp/js/query.spec.js

```diff
@@ -0,0 +1,333 @@
+/*=========================
+    RESTAURANT QUERY AND RE-ROLL
+ =========================*/
+describe("Test the query function", ()=>{
+    let dummyElement;
+    let htmlReponse;
+
+    beforeEach(() => {
+        htmlResponse = new Response('{\"pick\":{\"name\":\"Chipotle\",\"rating\":\"4.5\",\"photos\":[\"photo1\",\"photo2\"]},\"results\":[{\"name\":\"Qdoba\",\"rating\":\"4\",\"photos\":[\"photo1\",\"photo2\"]}],\"status\":\"OK\"}');
```

Comment:
would a backtick string literal be easier? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

also apply below

Reply:
response was created with a JSON generator so it wasn't too hard